test/unit/request-auth.test.js

```diff
@@ -73,6 +73,55 @@ describe('RequestAuth', function () {
         });
     });
 
+    it('should be able to construct with multiple options', function () {
```

Comment:
Not sure what this test is about 😕

Reply:
Removed the test had no impact on coverage.